python/tank/util/defaults_manager.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from tank_vendor import shotgun_authentication as sg_auth
+
+
+class CoreDefaultsManager(sg_auth.DefaultsManager):
+    """
+    This defaults manager implementation taps into core's shotgun.yml to provide
```

Comment:
All the docstrings in this class talk about `shotgun.yml`. I think we should remove references to shotgun.yml – this interface is a binding between the authentication layer and the specific host details for core – *how* those host details are retrieved is what we are trying to encapsulate with this class, so in my view, `shotgun.yml` becomes an internal concern of the class. This is good for encapsulation! Hope it makes sense - I know this is subtle, but I still wanted to point it out because it feels like an important distinction to make!